platform/broadcom/sonic-platform-modules-delta/agc032/modules/delta_agc032_qsfp.c

```diff
@@ -0,0 +1,819 @@
+#include <linux/device.h>
```

Comment:
Please use optoe driver rather than implementing a custom SFP driver

Reply:
Sorry for the confuse, 
this driver is not sfp driver, this driver is used to create qsfp eeprom device node,
like "/sys/bus/i2c/devices/xx-0050",
should i rename this file?